src/BrandEmbassyCodingStandard/Sniffs/Classes/ForbiddenStaticMethodSniff.php

```diff
@@ -90,11 +91,9 @@ private function addMethodCallError(
         int $staticMethodCallPointer,
         string $forbiddenClass
     ): void {
-        $methodNamePointer = $staticMethodCallPointer + 2;
-        $call = TokenHelper::getContent($phpcsFile, $staticMethodCallPointer, $methodNamePointer);
-        $expectedCall = TokenHelper::getContent($phpcsFile, $staticMethodCallPointer + 1, $methodNamePointer);
+        $call = TokenHelper::getContent($phpcsFile, $staticMethodCallPointer, $staticMethodCallPointer);
 
-        $errorMessage = sprintf('Using %s is forbidden, %s%s should be used.', $call, $forbiddenClass, $expectedCall);
+        $errorMessage = sprintf('Using %s:: is forbidden. Call %s:: directly', $call, $forbiddenClass);
```

Comment:
Hláška "Using self:: is forbidden. Call Foo:: directly" moc nedává smysl. Doplnil bych rozhodně volané metody.